css/style.css

```diff
@@ -556,11 +556,11 @@ figcaption details p {
     margin-bottom: 2em;
 }
 
-.floe .floe-image-UIO             { background: url("../images/floe-02.png") no-repeat 0 0 / 325px 325px; }
-.floe .floe-image-exploreTool     { background: url("../images/floe-01.png") no-repeat 0 0 / 325px 325px; }
-.floe .floe-image-cloud           { background: url("../images/floe-05.png") no-repeat 0 0 / 325px 325px; }
-.floe .floe-image-preferenceIcons { background: url("../images/floe-04.png") no-repeat 0 0 / 325px 325px; }
-.floe .floe-image-videoPlayer     { background: url("../images/floe-03.png") no-repeat 0 0 / 325px 325px; }
+.floe .floe-image-UIO             { background: url("../images/floe-02.png") no-repeat 0 0 / 290px 290px; }
+.floe .floe-image-exploreTool     { background: url("../images/floe-01.png") no-repeat 0 0 / 290px 290px; }
+.floe .floe-image-cloud           { background: url("../images/floe-05.png") no-repeat 0 0 / 290px 290px; }
+.floe .floe-image-preferenceIcons { background: url("../images/floe-04.png") no-repeat 0 0 / 280px 280px; }
+.floe .floe-image-videoPlayer     { background: url("../images/floe-03.png") no-repeat 0 0 / 280px 280px; }
```

Comment:
You didn't fix it for themed images. The problem persists when themes are changed. 
For Example:

1) 
![Screenshot from 2020-03-03 09-52-57](https://user-images.githubusercontent.com/34926285/75742853-006cc680-5d35-11ea-8410-4e339c8eb6b8.png)

2) 
![Screenshot from 2020-03-03 09-53-03](https://user-images.githubusercontent.com/34926285/75742860-0662a780-5d35-11ea-9393-743e54b3a269.png)